tests/unit_tests/test_tracks.py

```diff
@@ -168,7 +168,7 @@ def test_restart_track(run_in_tmpdir, sphere_model):
 
     # generate lost particle files
     with pytest.raises(RuntimeError, match='Maximum number of lost particles has been reached.'):
-        sphere_model.run(output=False)
+        sphere_model.run(output=False, threads=1)
```

Comment:
Would using more threads change the result?

Reply:
I've been seeing sporadic failures recently in CI from this test because the warning output from multiple threads doesn't necessarily come out sequentially, and the mixing of outputs was causing some weirdness with decoding output. Reducing to 1 thread doesn't reduce the purpose/value of this test but so far it does seem to have fixed the issues with sporadic failures